api/types/types.go

```diff
@@ -253,3 +254,13 @@ type ContainerConfig struct {
 	CpuShares  int64
 	Cpuset     string
 }
+
+// MountPoint represents a mount point configuration inside the container.
+type MountPoint struct {
+	Name        string `json:",omitempty"`
+	Source      string
+	Destination string
+	Driver      string `json:",omitempty"`
+	Mode        string
```

Comment:
Place a small doc comment for `Mode` that this might be known as `Relabel` internally.